### PR TITLE
Add parquet and json schemas for Health ping

### PR DESCRIPTION
**Pull request by katejim**
[Bug 1383730](https://bugzilla.mozilla.org/show_bug.cgi?id=1383730) and [Bug 1381783](https://bugzilla.mozilla.org/show_bug.cgi?id=1381783).
[Health ping docs.](http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/health-ping.html)

### CODE REVIEW

File: templates/telemetry/health/health.4.schema.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          "type": "object"
+        },
+        "pingDiscardedForSize": {
+          "properties": {
+            "additionalProperties": {
+              "type": "number"
+            }
+          },
+          "minProperties": 1,
+          "maxProperties": 10,
```
Comment (fbertsch) [inline templates/telemetry/health/health.4.schema.json]: Why have this? Seems like a footgun that could hurt us as we expand the number of types of pings we are sending. (iiuc, this is limiting the number of ping types that can be included, as `pingDiscardedForSize` is a map from `pingType` -> `numberDiscarded`)

Comment (katejim) [inline templates/telemetry/health/health.4.schema.json]: @fbertsch We decided to have Health ping minimal in size, that is why every time we want to send only top ten pings. But in sendFilure block, i  probably should remove maxPropertirs

Comment (katejim) [inline templates/telemetry/health/health.4.schema.json]: @fbertsch top ten pingTypes

Comment (fbertsch) [inline templates/telemetry/health/health.4.schema.json]: @katejim to summarize: client-side is guaranteed to only return the top ten ping types. Is this top ten per session (ordered by number of discarded pings), or top ten as a static list of pings we would include?

Comment (katejim) [inline templates/telemetry/health/health.4.schema.json]: @fbertsch In this block "pingDiscardedForSize" we have list of pairs **pingType**: **count**. Client size guaranteed, that when Health ping is submitted, "pingDiscardedForSize" list contains only top ten pairs. Health ping is submitted at most one in hour. See [docs](http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/health-ping.html)

File: schemas/telemetry/health/health.4.parquetmr.txt [hunk 2]
```diff
@@ -0,0 +1,48 @@
+message health {
+  required binary type (UTF8);
+  required binary id (UTF8);
+  required binary creationDate (UTF8);
+  required double version;
```
Comment (trink) [inline schemas/telemetry/health/health.4.parquetmr.txt]: version should be an integer type

File: templates/telemetry/health/health.4.parquetmr.txt [hunk 3]
```diff
@@ -0,0 +1,48 @@
+message health {
+  required binary type (UTF8);
+  required binary id (UTF8);
+  required binary creationDate (UTF8);
+  required double version;
```
Comment (trink) [inline templates/telemetry/health/health.4.parquetmr.txt]: should be an integer type

File: templates/telemetry/health/health.4.parquetmr.txt [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  required binary clientId (UTF8);
+  required group application {
```
Comment (trink) [inline templates/telemetry/health/health.4.parquetmr.txt]: this should include the predefined template https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/templates/include/telemetry/application.1.parquetmr.txt

File: templates/telemetry/health/health.4.parquetmr.txt (hunk 3 above)
Comment (katejim) [inline templates/telemetry/health/health.4.parquetmr.txt]: Thanks! I think I took double from [here](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/master/templates/telemetry/shield-study-addon/shield-study-addon.3.parquetmr.txt#L81).

**Pull request merged**